### PR TITLE
Ftr: Adding integration tests using dubbo-go-samples

### PR DESCRIPTION
**Pull request by EnableAsync**
**What this PR does**:
Adding integration tests using dubbo-go-samples

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by AlexStocks — APPROVED** [review]

File: .github/workflows/github-actions.yml [hunk 1]
```diff
@@ -69,9 +69,17 @@ jobs:
...
 
+      # This step only runs when the event type is a pull_request
     - name: Integrate Test
+      if: ${{ github.event_name == 'pull_request' }}
       run: |
-        chmod +x integrate_test.sh && ./integrate_test.sh ${{github.event.pull_request.head.repo.full_name}} ${{github.event.pull_request.head.sha}}
+        chmod +x integrate_test.sh && [[ -n "${{github.event.pull_request.head.repo.full_name}}" ]] && [[ -n "${{github.event.pull_request.head.sha}}" ]] && [[ -n "${{github.base_ref}}" ]] && ./integrate_test.sh ${{github.event.pull_request.head.repo.full_name}} ${{github.event.pull_request.head.sha}} ${{github.base_ref}}
+
+      # This step only runs when the event type is a push
+    - name: Integrate Test
```
Comment (LaurenceLiZhixin) [inline .github/workflows/github-actions.yml:79]: I guess change this name to 'Integrate Test With Dubbo-go-samples 1.5' would be better to distinguish two procedure

**Review by LaurenceLiZhixin — APPROVED** [review]

File: .github/workflows/github-actions.yml (hunk 1 above)
Comment (EnableAsync) [reply .github/workflows/github-actions.yml:79]: Does it mean to distinguish pull_request from push?

**Review by Mulavar — APPROVED** [review]

**Comment by AlexStocks** [comment]
@EnableAsync 我看还有一个问题你没回复，这是处理完了还是没处理完毕？

**Comment by EnableAsync** [comment]
> 
> 
> @EnableAsync 我看还有一个问题你没回复，这是处理完了还是没处理完毕？

处理完了

**Pull request merged**